From 7e32e5c716949a14f115a4122fddbe435860669d Mon Sep 17 00:00:00 2001
From: Nina Gertskin <ngertskin@habana.ai>
Date: Wed, 24 Feb 2021 11:29:17 +0200
Subject: [PATCH] [SW-0] scripts: fix release-test job path

release-test job moved to another location, so it path fixed accordingly

Change-Id: I110d6d193a44e5fa7bbe483faa0145841891a0ef
Signed-off-by: Nina Gertskin <ngertskin@habana.ai>
---

diff --git a/scripts/make/common/Makefile b/scripts/make/common/Makefile
index ea9a9d5..ad3f14a 100644
--- a/scripts/make/common/Makefile
+++ b/scripts/make/common/Makefile
@@ -95,7 +95,7 @@
 RELEASE_JOB_NAME := release-test
 RELEASE_JOB_BRANCH := "package_branch=$(RELEASE_BRANCH)"
 RELEASE_JOB_VER_COMPAT := "release_version=$(GEN_RELEASE_VER)"
-RELEASE_JOB_URL := https://sw-jenkins.habana-labs.com/job/$(RELEASE_JOB_NAME)/buildWithParameters
+RELEASE_JOB_URL := https://sw-jenkins.habana-labs.com/job/Embedded/job/CD/job/$(RELEASE_JOB_NAME)/buildWithParameters
 RELEASE_JOB_AUTH_USER := swembeddeduser
 RELEASE_JOB_AUTH_TOKEN := 11e05e25472cbe34be9a673ea71d2baa32
 RELEASE_JOB_AUTH := --auth-no-challenge --user $(RELEASE_JOB_AUTH_USER) --password $(RELEASE_JOB_AUTH_TOKEN)
